trajectory_point[] points
trajectory_point end_point
trajectory_point start_point
float32 z_height


